### ISSUE TITLE
I want to use in CHINESE!!!

### ISSUE THREAD

**Issue by 973xx** [spyder-ide/spyder#20292]
### What steps will reproduce the problem?

I want to use the application in CHINESE!!!!!!!!!!

### Traceback
```python-traceback
Traceback (most recent call last):
  File "F:\anaconda\lib\site-packages\spyder\plugins\preferences\widgets\configdialog.py", line 136, in button_clicked
    configpage.apply_changes()
  File "F:\anaconda\lib\site-packages\spyder\plugins\preferences\api.py", line 120, in apply_changes
    self.prompt_restart_required()
  File "F:\anaconda\lib\site-packages\spyder\plugins\preferences\api.py", line 834, in prompt_restart_required
    self.restart()
  File "F:\anaconda\lib\site-packages\spyder\plugins\preferences\api.py", line 838, in restart
    self.main.restart(close_immediately=True)
  File "F:\anaconda\lib\site-packages\spyder\app\mainwindow.py", line 1862, in restart
    self.application.restart(
  File "F:\anaconda\lib\site-packages\spyder\plugins\application\plugin.py", line 337, in restart
    console = self.get_plugin(Plugins.Console)
  File "F:\anaconda\lib\site-packages\spyder\api\plugins\new_api.py", line 428, in get_plugin
    raise e
  File "F:\anaconda\lib\site-packages\spyder\api\plugins\new_api.py", line 423, in get_plugin
    return self._main.get_plugin(plugin_name)
  File "F:\anaconda\lib\site-packages\spyder\app\mainwindow.py", line 161, in get_plugin
    raise SpyderAPIError(f'Plugin "{plugin_name}" not found!')
spyder.api.exceptions.SpyderAPIError: Plugin "internal_console" not found!
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.9.13
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.16.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.4.0 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.14.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.2.0 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  63.4.1 (OK)
sphinx >=0.6.6                :  5.0.2 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  23.2.0 (OK)

# Optional:
cython >=0.21                 :  0.29.32 (OK)
matplotlib >=2.0.0            :  3.5.2 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.4 (OK)
scipy >=0.17.0                :  1.9.1 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @973xx, thanks for reporting. This problem should be fixed in our latest version available in Anaconda (5.3.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
